node/rest/src/routes.rs

```diff
@@ -247,6 +248,44 @@ impl<N: Network, C: ConsensusStorage<N>, R: Routing<N>> Rest<N, C, R> {
         Ok(ErasedJson::pretty(mapping_value))
     }
 
+    // GET /<network>/program/{programID}/mapping/{mappingName}?all={true}&metadata={true}
+    pub(crate) async fn get_mapping_values(
+        State(rest): State<Self>,
+        Path((id, name)): Path<(ProgramID<N>, Identifier<N>)>,
+        metadata: Query<Metadata>,
+    ) -> Result<ErasedJson, RestError> {
+        // If the `all` query parameter is set, return the full mapping.
+        if metadata.all.unwrap_or(false) {
```

Comment:
nit: instead of having most of the body in a conditional block, this function could begin with the early error return that's currently at the very end, and then continue without the extra block

Reply:
Resolved with [87f3bbd](https://github.com/AleoNet/snarkOS/pull/3365/commits/87f3bbd57ef4540b9ce0f99bcac7e0ccfbb703b1)